test/object-store/sync/flx_migration.cpp

```diff
@@ -777,5 +798,87 @@ TEST_CASE("New table is synced after migration", "[flx][migration]") {
     }
 }
 
+TEST_CASE("Async open + client reset", "[flx][migration]") {
```

Comment:
Could we add a comment about what this tests?

Reply:
I think @michael-wb has a few tests which may replace/update this